service/app/controllers/usermanagement/validator/UserStatusRequestValidator.java

```diff
@@ -0,0 +1,28 @@
+package controllers.usermanagement.validator;
+
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.request.BaseRequestValidator;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+public class UserStatusRequestValidator extends BaseRequestValidator {
+
+  public void validateBlockUser(Request request) {
+    validateUserId((String) request.getRequest().get(JsonKey.USER_ID));
```

Comment:
Instead use validateParam (a method of BaseRequestValidator). Reference: SystemSettingsRequestValidator.java

Reply:
Use error code mandatoryParamsMissing.